reco_utils/dataset/movielens.py

```diff
@@ -268,11 +265,11 @@ def load_item_df(
 
 def _load_item_df(size, item_datapath, movie_col, title_col, genres_col, year_col):
     """Loads Movie info"""
-    item_header = []
-    usecols = []
-    if movie_col is not None:
-        item_header.append(movie_col)
-        usecols.append(0)
+    if title_col is None and genres_col is None and year_col is None:
```

Comment:
Looks like the check now is different from before? The previous checks the `movie_col` and return `None` if it does not exist, whilst now it checks other three columns.

Reply:
yeah, made `movie_col` as a default (must-have) column, checking earlier as: `if header is None or len(header) < 2:` (2 is the first two columns, user and movie), because there is no point to load only user ids.